plugin.rb

```diff
@@ -1,21 +1,21 @@
 # frozen_string_literal: true
 
-# name: discourse-plugin-office365-auth
-# about: Enable Login via Office365
+# name: discourse-plugin-microsoft365-auth
+# about: Enable Login via Microsoft 365
 # version: 0.0.1
 # authors: Matthew Wilkin
-# url: https://github.com/discourse/discourse-plugin-office365-auth
+# url: https://github.com/discourse/discourse-plugin-microsoft365-auth
 
 require 'auth/oauth2_authenticator'
 require File.expand_path('../omniauth-microsoft365.rb', __FILE__)
 
-enabled_site_setting :office365_enabled
+enabled_site_setting :microsoft_365_enabled
 
 class Office365Authenticator < ::Auth::OAuth2Authenticator
-  PLUGIN_NAME = 'oauth-office365'
+  PLUGIN_NAME = 'oauth-microsoft365'
 
   def name
-    'microsoft_office365'
+    'microsoft_365'
```

Comment:
Changing this will change the callback URL... so we either need to leave it, or build some way to maintain backwards compatibility